src/draw/sdl/lv_draw_sdl.c

```diff
@@ -58,13 +60,57 @@ static SDL_Texture * layer_get_texture(lv_layer_t * layer);
 /**********************
  *   GLOBAL FUNCTIONS
  **********************/
+static bool sdl_texture_cache_create_cb(cache_data_t * cached_data, void * user_data)
```

Comment:
It looks like an entry's callback rather than instance's callback.

Reply:
Its function is to provide instance with a callback to create entry.